packages/payment-proxy-client/src/App.tsx

```diff
@@ -357,34 +384,43 @@ export default function App() {
                         }
                         label="Skip payment"
                       />
-                      <FormControl>
-                        <RadioGroup
-                          name="availableFiles"
-                          row={true}
-                          value={selectedFile.url}
-                          onChange={(e) => {
-                            const found = files.find(
-                              (f) => f.url == e.target.value
-                            );
-                            if (found) {
-                              setSelectedFile(found);
-                            }
-                          }}
-                        >
-                          {files.map((file) => (
-                            <FormControlLabel
-                              value={file.url}
-                              key={file.url}
-                              control={<Radio />}
-                              label={
-                                file.fileName.length < 50
-                                  ? file.fileName
-                                  : "..." + file.fileName.slice(-50)
+                      <Box>
+                        <FormControl>
+                          <RadioGroup
+                            name="availableFiles"
+                            value={selectedFile.url}
+                            onChange={(e) => {
+                              const found = files.find(
+                                (f) => f.url == e.target.value
+                              );
+                              if (found) {
+                                setSelectedFile(found);
                               }
-                            />
-                          ))}
-                        </RadioGroup>
-                      </FormControl>
+                            }}
+                          >
+                            {files.map((file) => (
+                              <FormControlLabel
+                                value={file.url}
+                                key={file.url}
+                                control={<Radio />}
+                                label={
+                                  file.fileName.length < 50
+                                    ? file.fileName
+                                    : "..." + file.fileName.slice(-50)
+                                }
+                              />
+                            ))}
+                          </RadioGroup>
+                        </FormControl>
+                      </Box>
+                      {useMicroPayments && payDisabled && (
```

Comment:
If we remove `payDisabled` then the bar will always be displayed when use micropayments is checked.

Reply:
I am actually wondering now (given #1761) if we want the progress bar there all the time, and some other visual feedback to indicate the use of micropayments...